oragono.yaml

```diff
@@ -451,6 +451,12 @@ accounts:
         offer-list:
             #- "oragono.test"
 
+    # modes that are set by default when a user connects
```

Comment:
This is extra lame, but can you copy and paste this block into conventional.yaml (in the same place it appears in oragono.yaml)?

Reply:
Done.